src/pages/c/[configKind]/index.astro

```diff
@@ -12,11 +12,20 @@ import type { BreadCrumbList } from '../../../components/Navigation/BreadCrumb.a
 
 export async function getStaticPaths() {
     const configKinds = await getCollection("configKinds");
+    const renderer = new marked.Renderer();
+
+    // Custom render lists in the body
+    renderer.list = (body, ordered) => {
+      const type = ordered ? 'ol' : 'ul';
+      const className = ordered ? 'list-decimal' : 'list-disc';
```

Comment:
We should probably text-left lists too.

<img width="1219" alt="image" src="https://github.com/CircleCI-Public/Config.Tips/assets/33272306/54ac5538-4b3a-4c3a-8cca-6fbd87cf0352">

Reply:
I'll fix this up in #48 since it deals with that change (this PR is rebased on top of that branch)